specification/logs/semantic_conventions/Instrumentation/mobile-events.md

```diff
@@ -0,0 +1,46 @@
+# Semantic Conventions for mobile instrumentation
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for instrumentation on mobile platforms.
+
+<!-- toc -->
+
+- [Lifecycle instrumentation](#lifecycle-instrumentation)
+  * [`lifecycle.state` values](#lifecyclestate-values)
+    + [iOS](#ios)
+    + [Android](#android)
+
+<!-- tocstop -->
+
+## Lifecycle instrumentation
+
+This section defines how to apply semantic conventions when instrumenting application lifecycle.
+
+| Name              | Type   | Values             | Description                                |
+|-------------------|--------|--------------------|--------------------------------------------|
+| `event.name`      | String | `client.lifecycle` | The name of the event.                     |
+| `event.domain`    | String | `device`           | The domain of the event.                   |
+| `lifecycle.state` | String | platform specific  | The state entered at the time of the event |
```

Comment:
@bryce-b In https://github.com/open-telemetry/semantic-conventions/pull/30/files we are proposing that we consider this attribute as being part of the event payload. For the event payload the TC's recommendation is to use log_record's body field, but for the initial PoCs, we are keeping the payload in an attribute called `event.data`.

I suggest you follow similar structure and make `lifecycle.state` an item inside the event payload.

cc: @MSNev  @martinkuba

Reply:
I agree with @scheler, as part of defining an "event" all of the event properties should be in the `data` (which we are prototyping as in the `event.data`).

So in this case the value in `data` could be called just `state` as you have already defined the `domain` and `name`, this also would allow for future expansion of the event definition to include additional properties like (perhaps) `os` for example so that anyone querying the data could use this as a pivot.